### DISCUSSION TITLE
Newbie to homebrew - what is there in cask directory

### DISCUSSION THREAD

**Discussion by lorenzogithubdev** [Homebrew/discussions#4054] [Casks] [answered]
hi all, 

thank you in advance for the help, i would like to know what is there in this directory

/usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask/Casks/

and also why now i find in it a file called "viber.rb" even if i deleted the app. what can i do with this file ? can i delete it ?

thank you
regards

**Comment by SMillerDev** [comment, marked as answer]
See https://docs.brew.sh/Formula-Cookbook

It's the local version of the Homebrew/casks tap. It contains the install definitions for all software Homebrew caries.

**Reply by lorenzogithubdev** [reply]
thank you

**Comment by lorenzogithubdev** [comment]
I didn't get exactly what you mean .. can you please explain me better
thanks :)

**Reply by SMillerDev** [reply]
No, that's as good as it gets.

**Reply by lorenzogithubdev** [reply]
ohh thank you very much :)
